virttest/utils_test/__init__.py

```diff
@@ -251,6 +251,72 @@ def update_boot_option(
             session.close()
 
 
+def update_vm_default_kernel(
+    vm,
+    kernel_version,
+    reboot=True,
```

Comment:
I don't think `reboot` should have a default value. I saw it where it's used and `update_vm_default_kernel`, honestly I didn't expect it to do more than the config-update initially. IMO the user of this function must select a value explicitly because the reviewer of the code where this function is used might not know it and `reboot=True` or `False` tells them that this function also reboots. Reboot is such an important and time consuming task that I think it shouldn't be done silently.